README.md

```diff
@@ -8,3 +8,18 @@ Originally created by the [HPC team of Ghent University](http://ugent.be/hpc).
 
 `vsc-mympirun` is made available under the GNU General Public License
 (GPL) version 2.
+
+
+# Workflow
+
+The first step of mympirun is making sure that every mpirun command passes through mympirun. this is accomplished by prepending the mympirun path to $PATH.
```

Comment:
what you describe is the `fake mpirun`, what is prepended to PATH is the fake subdir (and the `mympirun path` is a filename, so adding that to PATH makes no sense either)